docs/src/developing/runtime-facilities/zk-token-proof.md

```diff
@@ -0,0 +1,126 @@
+---
+title: ZK Token Proof Program
+---
+
+The native Solana ZK Token proof program verifies a number of zero-knowledge
+proofs that are tailored to work with Pedersen commitments and ElGamal
+encryption over the elliptic curve
+[curve25519](https://www.rfc-editor.org/rfc/rfc7748#section-4.1). The program
+was originally designed to verify the zero-knowledge proofs that are required
+for the [SPL Token-2022](https://spl.solana.com/token-2022) program. However,
+the zero-knowledge proofs in the proof program can be used in more general
+contexts outside of Token-2022 as well.
+
+- Program id: `ZkTokenProof1111111111111111111111111111111`
+- Instructions:
+  [ProofInstruction](https://github.com/solana-labs/solana/blob/master/zk-token-sdk/src/zk_token_proof_instruction.rs)
+- Status: The program is in active development and is not yet available. Please
+  follow the feature gate `zk1snxsc6Fh3wsGNbbHAJNHiJoYgF29mMnTSusGx5EJ` for
+  updates.
```

Comment:
Assuming we don't backport this documentation to v1.16 it seems fine to remove this status information entirely, so we don't need to remember to update it later. This PR then ships in v1.17 and by that time the program should have been activated on mainnet

Reply:
Yep, sounds good!